lsif/src/server.ts

```diff
@@ -94,26 +94,127 @@ async function main(): Promise<void> {
     await ensureDirectory(path.join(STORAGE_ROOT, 'tmp'))
     await ensureDirectory(path.join(STORAGE_ROOT, 'uploads'))
 
+    // Create queue to publish jobs for worker
+    const queue = await setupQueue(logger)
+
+    setInterval(() => {
+        // Queue an update on a sleep loop until we have better heuristics
```

Comment:
```suggestion
        // Queue an update on an interval until we have better heuristics
```
"Sleep" doesn't really exist in Node :)